sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/Utility.java

```diff
@@ -129,19 +134,30 @@ static void validateNettyVersions(Consumer<String> logger) {
 
             // Version shouldn't be null as azure-core-http-netty has it as a dependency, but it could have been
             // excluded. Don't include it as a warning for native dependencies as it is optional.
-            if (version != null && !Objects.equals(version.artifactVersion(), nettyTcnativeVersion)) {
-                versionMismatches.add("'io.netty:" + artifact + "' version: " + version.artifactVersion()
-                    + " (expected: " + nettyTcnativeVersion + ")");
+            if (version != null) {
+                versionInformation.add("'io.netty:" + artifact + "' version: " + version.artifactVersion());
             }
         }
 
-        if (!versionMismatches.isEmpty()) {
-            logger.accept("The following Netty dependencies have versions that do not match the versions specified in "
-                + "the azure-core-http-netty pom.xml file. This may result in unexpected behavior. If your application "
-                + "runs without issue this message can be ignored, otherwise please update the Netty dependencies to "
-                + "match the versions specified in the pom.xml file. Versions found in runtime: "
-                + CoreUtils.stringJoin(",", versionMismatches));
+        String versionInformationString = CoreUtils.stringJoin(", ", versionInformation);
+        StringBuilder stringBuilder
+            = new StringBuilder().append("The following is Netty version information that was found on the classpath: ")
+                .append(versionInformationString)
+                .append(". ");
+
+        if (nonNativeNettyVersions.size() > 1) {
+            stringBuilder.append(NETTY_VERSION_MISMATCH_LOG);
         }
+
+        stringBuilder.append("The version of azure-core-http-netty being used was built with Netty version ")
+            .append(nettyVersion)
+            .append(" and Netty Tcnative version ")
+            .append(nettyTcnativeVersion)
+            .append(". If your application runs without issue this message can be ignored, otherwise please align the "
+                + "Netty versions you are using (this doesn't need to be the versions used by azure-core-http-netty if "
+                + "the versions of Netty being used are newer).");
```

Comment:
Whatever the final phrase is, please also include a matching update to the TSG / docs in the same PR to add a section so that it'll be discoverable when people search for the exact wording online.

Reply:
Actually can you embed an aka.ms link directly to the doc section on this?  This is what other SDKs do and is part of our SDK design guidelines now on exception messages (I get that this is info logging, but it seems to be something that users will need to read docs on)